packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -408,7 +408,10 @@ class ScaleGestureRecognizer extends OneSequenceGestureRecognizer {
   /// Typically this is the number of fingers being used to pan the widget using the gesture
   /// recognizer.
```

Comment:
Should the public docs mention anything about this?

Reply:
Sure, will mention trackpad gestures here